bedrock/mozorg/views.py

```diff
@@ -198,7 +199,14 @@ def get_context_data(self, **kwargs):
 
 
 def home(request):
-    return l10n_utils.render(request, 'mozorg/home/home.html')
+    locale = l10n_utils.get_locale(request)
+
+    if locale == 'en-US' and switch('experiment-home-q32017'):
```

Comment:
I'm sure there's a good reason, but why hide the new page behind a switch if it's going out to 100% en-US?

Reply:
I figured just in case it bombs in terms of engagement and we want to revert quickly.